File: doc/quote-azure-k8-backend/Models/QuotePageResponse.cs
namespace quote_azure_k8_backend.Models
{
    public class QuotePageResponse
    {
        public List<Quote> Quotes { get; set; } = new List<Quote>();
        public int TotalCount { get; set; }
        public int Page { get; set; }
        public int PageSize { get; set; }
        public int TotalPages { get; set; }
    }
}

File: k8-quote-api/src/main/java/com/quote/k8/dto/QuotePageResponse.java
package com.quote.k8.dto;

import com.quote.k8.model.Quote;
import java.util.List;

public class QuotePageResponse {
    public List<Quote> quotes;
    public int totalCount;
    public int page;
    public int pageSize;
    public int totalPages;

    // Constructors
    public QuotePageResponse() {}

    public QuotePageResponse(List<Quote> quotes, int totalCount, int page, int pageSize, int totalPages) {
        this.quotes = quotes;
        this.totalCount = totalCount;
        this.page = page;
        this.pageSize = pageSize;
        this.totalPages = totalPages;
    }
}
